e2e/tests/omnibus-sepolia.toml/bootstrap.js

```diff
@@ -9,13 +9,23 @@ Promise.resolve()
     return ensurePoolConfiguration([
       {
         marketId: extras.legacy_market_id,
-        weightD18: 99,
+        weightD18: 90,
         maxDebtShareValueD18: ethers.utils.parseEther('1'),
       },
       {
         marketId: extras.synth_usde_market_id,
         weightD18: 1,
         maxDebtShareValueD18: ethers.utils.parseEther('1'),
       },
+      {
+        marketId: 3,
```

Comment:
Why is market id hardcoded? 
We need to pull all ids from cannon state. If this market id is not present in cannon state in any way (like for example PerpsMarket emits an event with supermarket id in constuctor) - we then need to update BFP Contract to do so

Reply:
This was copied over from past config, I think it can be safely removed

Reply:
Actually, after removing it, I get `CapacityLocked` on the bootstrap step.

Reply:
Yea I dont know what market it is.. But at some point it was released. And if we dont inlcude it we get `CapacityLocked` as  I guess it has some collateral delegated to it

Reply:
Yeah was not suggesting removing it as markets are append only to the pool at this point.